### PR TITLE
Feature/gradle oci

### PR DESCRIPTION
**Pull request by SgtSilvio**
https://hivemq.kanbanize.com/ctrl_board/41/cards/18924/details/

**Motivation**

- Provide multi-arch container image
- Improve container image user handling

**Changes**

- Build multi-arch container image for platforms: linux/amd64, linux/arm64/v8
- Removed root step down, run directly as user 10000
- Removed `hivemq` user and group, only use numeric uid 10000
- Cleaned up GitHub workflows

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -220,6 +222,77 @@ val hivemqZip by tasks.registering(Zip::class) {
...
+    registries {
+        dockerHub {
+            credentials()
+        }
+    }
+    imageDefinitions.register("main") {
+        imageName.set("hivemq/hivemq-ce")
+        allPlatforms {
+            parentImages {
+//                add("library:eclipse-temurin:11.0.22_7-jre-jammy") { isChanging = true }
```
Comment (ymengesha) [inline build.gradle.kts:235]: let's remove this comment.

**Review by ymengesha — APPROVED** [review]
LGTM!

**Review by afalhambra-hivemq — COMMENTED** [review]
We should update the README file to specify how this works, it is still pointing out to a file which no longer exist after this PR was merged.

File: docker/build.sh [hunk 2]
```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (ymengesha) [reply docker/build.sh:1]: Good callout!

**Pull request merged**